protocol-definitions/custom/Custom.yaml

```diff
@@ -868,5 +868,44 @@ customTypes:
           type: Data
           nullable: true
           since: 2.0
-
-
+  - name: SqlError
+    since: 2.1
+    params:
+      - name: code
+        type: int
+        nullable: false
+        since: 2.1
+      - name: message
+        type: String
+        nullable: true
+        since: 2.1
+      - name: originatingMemberId
+        type: UUID
+        nullable: false
+        since: 2.1
+  - name: SqlColumnMetadata
+    since: 2.1
+    returnWithFactory: true
+    params:
+      - name: name
+        type: String
+        nullable: false
+        since: 2.1
+      - name: type
+        type: int
+        nullable: false
+        since: 2.1
+  - name: SqlRowMetadata
```

Comment:
if this is a single parameter custom type, you can unwrap it

Reply:
Will do.

Reply:
SqlRowMetadata has been removed.